docs/platforms/android/configuration/options.mdx

```diff
@@ -329,7 +335,7 @@ A boolean value that determines whether the app start process will be profiled.
 
 ## Legacy Profiling Options
 
-<Alert title="✨ Note">
+<Alert title="Note">
 
 SDK versions lower than 8.6.0 may use the legacy profiling using the following options.
```

Comment:
`l`: this is called out already (though slightly differently) above, make sure that this is intended